Cython/Compiler/Nodes.py

```diff
@@ -3691,10 +3691,13 @@ def generate_tuple_and_keyword_parsing_code(self, args, success_label, code):
 
         code.putln('{')
         all_args = tuple(positional_args) + tuple(kw_only_args)
-        code.putln("static PyObject **%s[] = {%s};" % (
-            Naming.pykwdlist_cname,
-            ','.join(['&%s' % code.intern_identifier(arg.name)
-                      for arg in all_args if not arg.pos_only] + ['0'])))
+        non_posonly_args = [arg for arg in all_args if not arg.pos_only]
+        do_generate_kw_unpacking = bool(non_posonly_args) or self.starstar_arg
```

Comment:
If you care about a `bool` here, then the whole expression on the right side should be wrapped in `bool(…)`.

Reply:
Done